### PR TITLE
Add swarmpit

### PR DESCRIPTION
**Pull request by jirkapenzes**
I added Swarmpit app into your awesome Docker tools.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -364,6 +364,7 @@ Services to securely store your Docker images.
 * [Rancher](https://github.com/rancher/rancher) - An open source project that provides a complete platform for operating Docker in production by [@rancher][rancher].
 * [Serf](https://github.com/hashicorp/serf) - Service orchestration and management tool by [@hashicorp][hashicorp]
 * [Shipyard](https://github.com/shipyard/shipyard) - Composable Docker Management
+* [Docker Swarm](https://github.com/docker/swarm) (Swarm: a Docker-native clustering system)
```
Comment (veggiemonk) [inline README.md]: ??? swarmpit ???

**Comment by jirkapenzes** [comment]
@veggiemonk Thanks for review and sorry. That was bad commit. Could you please check PR again? Thanks.

File: README.md [hunk 2]
```diff
@@ -364,6 +364,7 @@ Services to securely store your Docker images.
 * [Rancher](https://github.com/rancher/rancher) - An open source project that provides a complete platform for operating Docker in production by [@rancher][rancher].
 * [Serf](https://github.com/hashicorp/serf) - Service orchestration and management tool by [@hashicorp][hashicorp]
 * [Shipyard](https://github.com/shipyard/shipyard) - Composable Docker Management
+* [Swarmpit](http://swarmpit.io/) - Lightweight Docker Swarm orchestration. Swarmpit provides clean way to manage your Docker Swarm cluster with various handful features such Service management, Smart search, Shared access, Private registries and more to come soon.
```
Comment (veggiemonk) [inline README.md]: the url should be the project on github rather than the websites

Comment (veggiemonk) [inline README.md]: Another thing, no need for capital letter at the begining of each work.
```
Lightweight Docker Swarm orchestration. Swarmpit provides clean way to manage your Docker Swarm cluster with various handful features such service management, smart search, shared access, private registries.
```
I removed the "more to come soon" as you can come back and add them here when the features are ready :D

**Comment by jirkapenzes** [comment]
Thanks. I push new commit with changes ;)

**Comment by veggiemonk** [comment]
@jirkapenzes Good work! Thanks for your contribution. It looks like a nice project!
If we can help you in any way, let us know!

Awesome-docker team

**Comment by jirkapenzes** [comment]
@veggiemonk Big thanks! Would we love to get feedback about our app. If you have time or if you know someone who have time please try it :) Feel free for feedbacks ;)

**Pull request merged**